m3/build.gradle

```diff
@@ -42,16 +41,3 @@ compileThrift {
         project.hasProperty('genThrift')
     }
 }
-
-jmh {
```

Comment:
why remove this? does the `m3` module inherit the jmh settings from the parent?

Reply:
There's no benchmarks in this project